frontend/src/routes/project/[project_code]/+page.ts

```diff
@@ -1,6 +1,10 @@
 import { getClient } from "$lib/graphQLClient";
 import type { PageLoadEvent } from "./$types";
 import { graphql } from "$lib/gql";
+import type { ProjectPageQuery } from "$lib/gql/graphql";
```

Comment:
do you think it would be helpful to have the types in `$lib/graphQLClient` or vice-versa?  I'm assuming consumers will almost always have to import both but maybe that's not correct?